src/util/math.js

```diff
@@ -43,15 +28,10 @@ const binomialLog = (n, k) => {
 // K - number of successes in population
 // n - number of items in sample
 // N - number of items in population
-const hyperGeometricTest = (k, K, n, N) => {
-  let sum = 0;
-  for (let i = 0; i < k; i++) {
-    sum += Math.exp(
-      binomialLog(K, i) + binomialLog(N - K, n - i) - binomialLog(N, n)
-    );
-  }
-  return sum;
-};
+export const hyperGeometricTest = (k, K, n, N) => cdf(k - 1, N, K, n);
```

Comment:
There must be a reason why `k` is changed to `k-1` inside cdf invocation. It will be nice to find it out and write it as a comment.

Reply:
Here is what I've found so far:

You also have to do `k-1` in `R`, if you use `fisher.test` with `alternative="less"`:
```
k <- 1; K <- 5; n <- 10; N <- 50; k <- k-1; mat <- matrix(c(k, K-k, n-k, N-K-n+k), nrow=2, ncol=2); print(1 - fisher.test(mat, alternative="less")$p.value, digits = 20);
```
But you don't need to with `alternative="greater"`:
```
k <- 1; K <- 5; n <- 10; N <- 50; mat <- matrix(c(k, K-k, n-k, N-K-n+k), nrow=2, ncol=2); print(mat); fisher.test(mat, alternative="greater")$p.value;
```

Reply:
Also this is how I got the fisher table:
![image](https://user-images.githubusercontent.com/8326331/75375810-df235a80-589c-11ea-9e94-3ea4d1cf2b69.png)

Where `N`, `K`, `n`, and `k` are known:
```
N = geneList.length;
K = selectedGenes.length;
n = signature.participatingGenes.length;
k = signature.selectedParticipatingGenes.length;
```

Reply:
I think it comes down to this. What we really want (as determined by Casey and the people who use the software) is circled in green, but what the `stdlib` `cdf` function gives us by default (with no option to choose anything else) is circled in blue, therefore we have to subtract 1 from `k` to get the desired result:
![image](https://user-images.githubusercontent.com/8326331/75378470-bd78a200-58a1-11ea-8daa-59961b766b5e.png)

Reply:
@dongbohu i believe the new commits with comment updates addresses this.